modules/integration/src/test/java/io/ballerina/messaging/broker/integration/standalone/cli/PrintStreamHandler.java

```diff
@@ -30,35 +30,52 @@
  */
 public class PrintStreamHandler {
 
-    private static ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-    private static PrintStream printStream = new PrintStream(outputStream);
+    private static ByteArrayOutputStream errStream = new ByteArrayOutputStream();
+    private static PrintStream errPrintStream = new PrintStream(errStream);
     private static PrintStream originalErrorStream = System.err;
 
+    private static ByteArrayOutputStream outStream = new ByteArrayOutputStream();
+    private static PrintStream outPrintStream = new PrintStream(outStream);
+    private static PrintStream originalOutStream = System.out;
```

Comment:
Maybe we can define a method with this logic since this duplicates what we have done in line 33-35.